docker/registry.Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:20.04
```

Comment:
You could also think of using something like: https://github.com/tiangolo/uwsgi-nginx-docker

Reply:
I guess I could, I've never used uWSGI. I've looked at the documentation a bit, but I can't find whether it uses processes or threads and how to configure that, other than that it seems to use two processes by default. Since the Python code contains the registry state it cannot run in a multiprocess model. (We'd need a real database backend to make that work, but I haven't implemented that.) So too much effort for now, but I'll keep it in mind.